onoff.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const fs = require('fs');
+const path = require('path');
```

Comment:
The quality and maintenance statistics published on npm for `path` are both below 50%. See https://www.npmjs.com/search?q=path

Reply:
i wonder how those stats come about.   "path" is part of NodeJS.  curious :)